apicode/newapi.py

```diff
@@ -0,0 +1,100 @@
+import requests
+import inquirer
+import textwrap
+import os
+import texttable as tt
+from inquirer.themes import GreenPassion
+
+
+class NewsApi:
+    # function for picking the news source from the user
+    def get_user_source(self):
+        questions = [
+            inquirer.List('sources',
+                          message="From which news source would you like to acquire your headlines?",
+                          choices=['nbc-news', 'cnbc',
+                                   'bbc-news', 'cnn'],
+                          ),
+
+
+        ]
+        # theme adds a green background to the items in the list
+        answers = inquirer.prompt(questions, theme=GreenPassion())
+        print("News Source : ", answers['sources'])
+
+        return answers['sources']
+    # function to retrieve a product key for the environment variable
+
+    def get_api_key(self):
+        try:
```

Comment:
This is unneccessary. The code in except will never be executed, The get method returns none if no key is found